dags/public-health/covid19/jhu-county-to-esri.py

```diff
@@ -16,6 +17,7 @@
 TIME_SERIES_FEATURE_ID = "4e0dc873bd794c14b7bd186b4b5e74a2"
 JHU_FEATURE_ID = "628578697fb24d8ea4c32fa0c5ae1843"
 MSA_FEATURE_ID = "b37e229b71dc4c65a479e4b5912ded66"
+max_record_count = 2_000_000
```

Comment:
Ooh, an underscore literal in the wild!

Reply:
Do you know how high we can make this? Hopefully this should be enough (unless this COVID thing goes *much* longer), but this would run out eventually.

Reply:
Up it to 6M